### PR TITLE
Add support to verify and get alert text

### LINKED ISSUE #440: Assert future dialogs
Converting a couple of projects using selenium library to robotframework-browser.

One of the things that can't be converted is checking the text within a dialog.

With browser it is possible to accept or decline a future dialogs, but it would be nice to have the possibility to check the text within the dialog. Same as in selenium library. When te future dialogs doesn't contain the expected text, the test should fail.

### PR DESCRIPTION
**Pull request by aaltat**
Fixes #440

### CODE REVIEW

File: node/playwright-wrapper/interaction.ts [hunk 1]
```diff
@@ -141,13 +141,30 @@ export async function handleAlert(request: Request.AlertAction, page: Page): Pro
...
     page.on('dialog', fn);
     return emptyWithLog('Set event handler for next alert');
 }
 
+export async function waitForAlert(request: Request.AlertAction, page: Page): Promise<Response.String> {
+    const alertAction = request.getAlertaction() as 'accept' | 'dismiss';
+    const promptInput = request.getPromptinput();
+    const dialogObject = await page.waitForEvent('dialog');
+    const message = dialogObject.message();
+    if (alertAction == 'accept' && promptInput) {
```
Comment (mkorpela) [inline node/playwright-wrapper/interaction.ts:158]: Please use `===`

File: node/playwright-wrapper/interaction.ts [hunk 2]
```diff
@@ -141,13 +141,30 @@ export async function handleAlert(request: Request.AlertAction, page: Page): Pro
...
+        dialogObject.accept(promptInput);
+    } else if (alertAction == 'accept') {
```
Comment (mkorpela) [inline node/playwright-wrapper/interaction.ts:160]: Please use `===`

**Review by mkorpela — CHANGES_REQUESTED** [review]
Equals in js is the shit. Use === instead of ==

**Comment by emakaay** [comment]
@aaltat  👍  looking good !

**Comment by aaltat** [comment]
@emakaay  this is not merged to master. If you want, you can install directly from GitHub repo with pip and give it a try.

**Comment by emakaay** [comment]
Thanks Tatu,
I'm already using this feature, and it works brilliantly.

Greets,
Erik

**Pull request merged**